amplify/backend/api/amplifyDatasource/schema.graphql

```diff
@@ -2,6 +2,7 @@ type Restaurant @model {
   id: ID!
   name: String!
   rating: Int
+  photo: String!
```

Comment:
Are you just storing the photo encoding as a string?

Reply:
Not really sure on what to store it as, so just stored it as is.

Reply:
Ok I'm just scared on how front-end is gonna deal with this, since we have to convert the string back into an image somehow.